### PR TITLE
Simpler setting value access

### PR DESCRIPTION
**Pull request by pythongosssss**
Updated clear check to use this

### CODE REVIEW

File: web/scripts/ui.js [hunk 1]
```diff
@@ -289,6 +282,16 @@ class ComfySettingsDialog extends ComfyDialog {
 				return element;
 			},
 		});
+
+		const self = this;
```
Comment (missionfloyd) [inline web/scripts/ui.js:286]: What's this for?

Comment (pythongosssss) [reply web/scripts/ui.js:286]: So if the getter/setters are called with a different context of `this`, it stores the reference to the current `this` as `self` and use `self` as the correct `this`, (i know ive phrased that badly... hopefully it makes sense) an example:

```js
  class X {
    #test;
    test() {
      this.#test = 123;
      var self = this;
      return {
        get useSelf() {
          return self.#test;
        },
        get useThis() {
          return this.#test;
        },
      }
    }
  }

  console.log(new X().test().useSelf);
  console.log(new X().test().useThis);
```

useSelf will output 123, useThis will crash

File: web/scripts/ui.js [hunk 2]
```diff
@@ -410,6 +413,13 @@ export class ComfyUI {
 			this.history.update();
 		});
 
+		const confirmClear = this.settings.addSetting({
+			id: "Comfy.ConfirmClear",
+			name: "Require confirmation when clearing workflow",
+			type: "boolean",
+			defaultValue: true,
```
Comment (comfyanonymous) [inline web/scripts/ui.js:420]: On a clean browser I see the checkbox ticked but it doesn't respect this value because it doesn't ask for confirmation.

Comment (pythongosssss) [reply web/scripts/ui.js:420]: Added missing default value

**Pull request merged**